lib/http-server.js

```diff
@@ -54,7 +54,9 @@ function HttpServer(options) {
       ? 'html'
       : options.ext;
   }
-
+if(this.gzip) {
+    this.headers['Content-Encoding'] = 'gzip, deflate';
```

Comment:
@gideonthomas  Fixed the errors, if you need anything else changed please let me know!